db/migrations/state/0001.sql

```diff
@@ -130,7 +130,7 @@ CREATE TABLE state.proof
     input_prover VARCHAR,
     prover VARCHAR,
     generating BOOLEAN DEFAULT FALSE,
-    PRIMARY KEY (batch_num, batch_num_final)    
+    PRIMARY KEY (batch_num, batch_num_final)
```

Comment:
The idea of the migrations files is to put new changes on new files, so nodes that are already running get their schema updated once they have the version that includes this code